templates/netlify/remix.init/edge-server.js

```diff
@@ -0,0 +1,8 @@
+// Import path interpreted by the Remix compiler
+import * as build from "@remix-run/dev/server-build";
+import { createRequestHandler } from "@remix-run/netlify-edge";
```

Comment:
Shouldn't we have both `@remix-run/dev` & `@remix-run/netlify-edge` in our `dependencies` when choosing for `"Netlify Edge Functions"`? 🤔